Mollier/SAM.Core.Mollier.UI/Modify/AddLinesSeries.cs

```diff
@@ -0,0 +1,102 @@
+﻿using SAM.Geometry.Planar;
+using System.Windows.Forms.DataVisualization.Charting;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SAM.Core.Mollier.UI
+{
+    public static partial class Modify
+    {
+
+        public static void AddLinesSeries(this Chart chart, MollierControlSettings mollierControlSettings)
```

Comment:
Return Series -> List<Series>
Move to SAM.Core.Mollier.Create
Rename to Series. Example below:

namespace SAM.Core.Mollier.UI
{
	public static partial class Create
	{
		public static List<Series> Series(this Chart chart, MollierControlSettings mollierControlSettings)
		{
			//Add code here
		}
	}
}